package.json

```
@@ -25,7 +25,7 @@
     "test-types": "tsc -p types/tests/"
   },
   "peerDependencies": {
-    "chart.js": "^3.2.0"
+    "chart.js": "^3.2.0 || 4.0.0"
```

I believe that wouldn't let you use it with any other versions of 4.x like 4.0.1 . I think you probably want:
```suggestion
    "chart.js": ">=3.2.0"
```

---

Sorry, did this in a rush. I wanted to use `^3.2.0 || ^4.0.0"` to be more strict and prevent this breaking in the future, but can update to `>=3.0.0` if you prefer